src/frontend/context/BrevModulContext.ts

```diff
@@ -118,18 +127,37 @@ const [BrevModulProvider, useBrevModul] = createUseContext(() => {
         avhengigheter: { brevmal },
     });
 
+    const fritekster = useFelt<FeltState<IFritekstFelt>[]>({
+        verdi: [],
+        valideringsfunksjon: (felt: FeltState<FeltState<IFritekstFelt>[]>) => {
+            return felt.verdi.some(
+                fritekst =>
+                    fritekst.valideringsstatus === Valideringsstatus.FEIL ||
+                    fritekst.verdi.tekst.length === 0
+            )
+                ? feil(felt, '')
```

Comment:
Mangler feilmelding?

Reply:
Virker som det er gjort slik fordi hver fritekst er også et Felt. Så f eks hvis man legger til en fritekst, men ikke skriver noe i det, så blir feilmeldingen til feltet vist, men ikke en generell feilmelding for alle fritekstene som en helhet. funksjonen lagInitiellFritekst ser det ut til at gjør dette. Men jeg skal sjekke det opp litt mer 👍